src/components/Write.js

```diff
@@ -66,45 +66,47 @@ class Write extends Component {
     const { isRead, icon, handleCloseClick } = this.props
 
     return (
-      <div className='ui centered card'>
-        <div className='content'>
-          <div className='ui form'>
-            <div className='field'>
-              <textarea
-                rows='10'
-                maxLength='200'
-                onChange={this.handleTextChange}
-                placeholder={
-                  isRead
-                    ? 'Write your response here! ✎'
-                    : 'Write your letter here! ✎'
-                }
-                value={this.state.content}
-              />
+      <>
+        <div className='eight wide centered column'>
+          <div className='ui fluid card'>
+            <div className='content'>
+              <div className='ui form'>
```

Comment:
Usually I think the form class is supposed to be use for form tags

https://semantic-ui.com/collections/form.html